Documentation/ceph-nfs-crd.md

```diff
@@ -442,3 +226,101 @@ configuration objects.
 
 Of note, it is possible to pre-populate the NFS configuration and export objects prior to starting
 NFS servers.
+
+
+## Known issues
+### server.active count greater than 1
+* Active-active scale out does not work well with the NFS protocol. If one NFS server in a cluster
+  is offline, other servers may block client requests until the offline server returns, which may
+  not always happen due to the Kubernetes scheduler.
+  * Workaround: It is safest to run only a single NFS server, but we do not limit this if it
+    benefits your use case.
+
+### Ceph v16.2.0 through v16.2.6
+* Ceph NFS management is buggy in Ceph Pacific v16.2.0 through v16.2.6.
```

Comment:
How about we just say we don't support these versions? But we can link to the v1.8 docs if they want to try.
```suggestion
* Ceph NFS management is not supported in Ceph Pacific v16.2.0 through v16.2.6.
```

Reply:
Removed this known issue and added the info to the top-of-doc warning